src/components/User/User.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './User.scss';
+
+import { UserProps } from '../../constants/proptypes';
+
+const User = ({ user }) => {
+  const { name, email, address } = user;
+
+  return (
+    <div className="user">
+      <p className="user--text">
+        <span className="user__text">By</span>
+        {name && (
```

Comment:
what if there is no `name`, you will have just `By` on the page,
place it inside condition as well